src/commands/change-case-create.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+# How to author commands: https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-commands
+description: >
+  create a change case.  Reqires install-change-case-mgmt to have been run
+
+parameters:
+  configuration_item:
+    description: Matches a GUS configuration item (example 'NPM')  The first part of the path is hard-coded to 'Salesforce.SF_Off_Core.DeveloperTools'.
```

Comment:
Why should we hardcode the path? I feel like it is more clear to just have the whole path passed in.

Reply:
Can this be passed in by the environment variable too?

Reply:
It was that way in the plugin--I just made a note of it.  Easy to change, probably more useful outside DevTools that way.

Reply:
updated in latest